model/src/main/kotlin/Identifier.kt

```diff
@@ -131,7 +131,7 @@ data class Identifier(
     fun toPurl() = "".takeIf { this == EMPTY }
         ?: buildString {
             append("pkg:")
-            val purlType = getPurlType()?.toString() ?: type.toLowerCase()
+            val purlType = getPurlType().toString()
```

Comment:
`getPurlType()` now returns a `String`, so `toString()` can be dropped here.

Reply:
Removed the variable declaration as it is no longer required.